site/en/blog/fledge-api/index.md

```diff
@@ -383,13 +422,13 @@ days. Successive calls overwrite previously stored values.
    want to add a browser to an interest group for a campaign that isn't running yet, or for some
    other future use, or they may temporarily have run out of advertising budget.
 
-\*\* In the current implementation of FLEDGE, `biddingLogicUrl`, `dailyUpdateUrl` and
-`trustedBiddingSignalsUrl` must have the same origin as owner. That may not be a long-term
-constraint, and the `ads` and `adComponents` URLs have no such constraint.
+\*\* In the current implementation of FLEDGE, `biddingLogicUrl`, `biddingWasmHelperUrl`,
+`dailyUpdateUrl` and `trustedBiddingSignalsUrl` must have the same origin as owner. That may not be
+a long-term constraint, and the `ads` and `adComponents` URLs have no such constraint.
 
 {: #update-interest-group}
 
-#### Update the interest group
+#### Update interest group attributes
 
 `dailyUpdateUrl` specifies a web server that returns JSON defining interest group properties,
 corresponding to the interest group object passed to `navigator.joinAdInterestGroup()`. This
```

Comment:
(Applies to line 436 and below -- Github doesn't let me comment there)

The current implementation link can be updated to point to the newer code: https://source.chromium.org/chromium/chromium/src/+/main:content/browser/interest_group/interest_group_storage.cc;l=671;drc=5a102f146faa0c21eb9cf255ceb46b35a158ab3f

(We could have the code search point to the symbol name, but I'm worried about link breakage if the symbol name changes.)

Also, the following are now supported for updates: priority.

biddingWasmHelperUrl and adComponents probably should be getting updated too (part of the code needed to update them is missing) -- I'll mail a change to make sure they're getting updated too.

Reply:
Thanks — I've updated the link, and added `biddingWasmHelperUrl ` to the list of attributes that can be changed.

> Also, the following are now supported for updates: priority.

What does this refer to?

Reply:
Actually, we don't support updating biddingWasmHelperUrl or  adComponents, but we should -- that's a bug that we should fix, and then update the documentation after that :)

We do support updating priority.

Reply:
Got it -- removed `biddingWasmHelperUrl` and `adComponents` for the moment.

I can see `priority` [in the code](https://source.chromium.org/chromium/chromium/src/+/main:content/browser/interest_group/interest_group_storage.cc;l=671;drc=5a102f146faa0c21eb9cf255ceb46b35a158ab3f) but not in the explainer. Is usage detailed somewhere?

Reply:
@brusshamilton priority is new -- it's used when setting a limit on the number of auction participants in the auction config, via perBuyerGroupLimits, which is a dictionary from buyer origin to the max number of auctions for that buyer -- if a star, '*', is used as a buyer, it sets the default for other unspecified buyers. 

At auction time, we pick a number of interest groups per buyer, according to the perBuyerGroupLimits limit, and we pick the highest priority interest groups (we sort the list by priority, and pick the top n; I think ties are broken arbitrarily).

Reply:
+1 to caraitto's description of priority.
Only thing to clarify is that ties are broken by choosing the interest group with expiration time farther in the future.

Reply:
Thanks both. 

Will `perBuyerGroupLimits` be documented in the explainer? 

> ties are broken

Excuse my ignorance — what does this refer to?

Reply:
We should probably add it to the explainer. I'll work on putting together a pull request for that.

> > ties are broken
> Excuse my ignorance — what does this refer to?
When we limit the interest groups we pick the highest priority ones first. But if interest groups have the same priority (i.e. they are tied), then we prefer groups with later expiration times.

Reply:
> ties are broken
>> Excuse my ignorance — what does this refer to?
> > > When we limit the interest groups we pick the highest priority ones first. But if interest groups have the same priority (i.e. they are tied), then we prefer groups with later expiration times.

Ah! Got it. I was imaging some abstruse CS concept :).